### PR TITLE
[MBL-16996][Parent] Score to letter grade conversion

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: test assignment list and assignment details with and without quantitative data restriction

refs: MBL-16996
affects: Parent
release note: Added letter grade conversion by grading scheme.

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode
- [x] A11y checked

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

File: apps/flutter_parent/lib/models/grade_cell_data.dart [hunk 1]
```diff
@@ -144,7 +150,9 @@ abstract class GradeCellData implements Built<GradeCellData, GradeCellDataBuilde
           ..state = GradeCellState.gradedRestrictQuantitativeData
           ..graphPercent = 1.0
           ..accentColor = accentColor
-          ..score = submission.grade
+          ..score = assignment.isGradingTypeQuantitative()
+              ? course.convertScoreToLetterGrade(submission.score, assignment.pointsPossible)
```
Comment (tamaskozmer) [inline apps/flutter_parent/lib/models/grade_cell_data.dart:154]: Do we need this check here? We already convert and assign the grade at row 131.

**Review by tamaskozmer — APPROVED** [review]

**Comment by tamaskozmer** [comment]
QA + 1

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Pull request merged**